src/tools/pbs_sleep.c

```diff
@@ -70,13 +79,23 @@ main(int argc, char *argv[])
 		exit(1);
 	}
 
+	#ifndef WIN32
+	signal(SIGINT, handler);
+	signal(SIGHUP, handler);
+	signal(SIGTERM, handler);
```

Comment:
You shouldn't need this one.  It should just die on its own.  The default action on a SIGTERM is to terminate.